dapp-example/tests/e2e/specs/index.spec.js

```diff
@@ -131,7 +142,7 @@ describe('Test Core Hedera User Scenarios', function() {
     }).timeout(180000);
 
     it('Transfer HBARs', function() {
-      cy.get('#sendHbarsToField').clear().type('0x54C51b7637BF6fE9709e1e0EBc8b2Ca6a24b0f0A');
+      cy.get('#sendHbarsToField').clear().type(randomHollowAccountAddress);
```

Comment:
`randomHollowAccountAddress` is undefined here

Reply:
Fixed